contracts/protocol/libs/FundsLib.sol

```diff
@@ -186,6 +193,11 @@ library FundsLib {
 
                     payoff.buyer = payoff.buyer + applyPercent(lastPrice, dispute.buyerPercent);
                     payoff.seller = payoff.seller + offerPrice - applyPercent(offerPrice, dispute.buyerPercent);
+
+                    // DR is always paid for Decided, but for Resolved it is paid only if escalated
+                    if (disputeState == BosonTypes.DisputeState.Decided || buyerEscalationDeposit > 0) {
```

Comment:
I think it's enough to check only that `buyerEscalationDeposit > 0`.

Technically, yes, it's possible that `buyerEscalationDeposit == 0` and `disputeState == BosonTypes.DisputeState.Decided`, but in that case `feeAmount` will also be 0. (`buyerEscalationDeposit` is calculated as some percentage of `feeAmount`)